notebooks/en/Prompt_Tuning_PEFT.ipynb

```diff
@@ -0,0 +1,7324 @@
+{
```

Comment:
<p>Same here, if you place the last line in a separate cell, we can show the output of the display, and remove the output with downloading</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/33/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>